CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # PyPRT ChangeLog
 
+## v1.4.0 (2021-07-XX)
+
+### Removed
+* Removed MacOS support

```

Comment:
We should add a small explanation: "Removed macOS support as PRT 2.4 and later is not available anymore on that platform."